__tests__/database/schema/ChatSchema.test.ts

```diff
@@ -482,6 +513,7 @@ describe('TESTING ChatSchema', () => {
         'userId',
         'channelId',
         'message',
+        [],
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `attachments` field should be a string that contains serialized JSON.
```diff
-  [],
+  '[]',
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->